src/test/java/com/wikia/webdriver/common/logging/Log.java

```diff
@@ -263,6 +273,8 @@ public static void stacktrace(Throwable throwable) {
 
   public static void stop() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this method "stop" is 12 which is greater than 10 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)